src/persist/src/persist.proto

```diff
@@ -70,3 +69,45 @@ message ProtoU64Description {
     ProtoU64Antichain upper = 2;
     ProtoU64Antichain since = 3;
 }
+
+message ProtoBatchInline {
+    ProtoBatchFormat format = 1;
+    oneof batch_type {
+        // To save bytes, only the seqno_lower and seqno_upper fields are set on
+        // this.
+        ProtoUnsealedBatchMeta unsealed = 2;
+        // To save bytes, only the desc field is set on this.
+        ProtoTraceBatchMeta trace = 3;
+    }
+}
+
+enum ProtoBatchFormat {
+    Unknown = 0;
+    // Arrow, which we'd use for the local blob cache if we use it, gets a
+    // structure like `[(K, V, T, D)]` so that we could mmap it and use it
+    // direction as our in-mem batches (which have this structure).
```

Comment:
nit: I think you meant "use it directly as our ..."

Reply:
Done